app/controllers/admin/manual_ptime_sync_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Admin::ManualPtimeSyncController < CrudController
+  self.nesting = :admin
+  before_action :render_unauthorized_not_admin
+  before_action :redirect_admin_unless_use_ptime_sync
```

Comment:
The naming of this method already contains logic that you can write out and put the conditional right here.
So instead of writing
` before_action :redirect_admin_unless_use_ptime_sync`
it could be something like this pseudocode
`before_action :redirect_admin, unless: { use_ptime_sync }`

This is not only more readable but also puts the logic where it belongs.
[docs](https://apidock.com/rails/v7.0.0/AbstractController/Callbacks/ClassMethods/before_action)